generics/services/sunbird.js

```diff
@@ -217,95 +202,120 @@ function callToSunbird(token, requestBody, url,type ="") {
-  * @param requestBody - requestBody .
+  * @param searchDetails - search details.
   * @param token - Logged in user token.
   * @returns {JSON} - All users data.
 */
 
-var searchOrganisation = function (requestBody, token) {
+const searchOrganisation = function (searchDetails, token) {
     return new Promise(async (resolve, reject) => {
 
         const searchOrgUrl =
-            process.env.sunbird_url + constants.endpoints.SUNBIRD_SEARCH_ORG;
-        let response = await callToSunbird(token, requestBody, searchOrgUrl);
+            process.env.SUNBIRD_URL + constants.endpoints.SUNBIRD_SEARCH_ORG;
+        let response = await callToSunbird(token, searchDetails, searchOrgUrl);
         return resolve(response);
 
-
-    })
+    });
 }
 
 /**
   * For creating organisation
   * @function
   * @name createOrganisation
-  * @param requestBody - requestBody .
+  * @param organisationDetails - organisation details.
   * @param token - Logged in user token.
   * @returns {JSON} - All users data.
 */
 
-var createOrganisation = function (requestBody, token) {
+const createOrganisation = function (organisationDetails, token) {
     return new Promise(async (resolve, reject) => {
 
-
         const searchOrgUrl =
-            process.env.sunbird_url + constants.endpoints.SUNBIRD_CREATE_ORG;
-        let response = await callToSunbird(token, requestBody, searchOrgUrl);
+            process.env.SUNBIRD_URL + constants.endpoints.SUNBIRD_CREATE_ORG;
+        let response = await callToSunbird(token, organisationDetails, searchOrgUrl);
         return resolve(response);
 
-
-    })
+    });
 }
 
-var updateOrganisationDetails  = function (requestBody, token) {
+
+/**
+  * For updating organisation details
+  * @function
+  * @name updateOrganisationDetails
+  * @param organisationDetails - organisation details .
+  * @param token - Logged in user token.
+  * @returns {JSON} - return updated organisation details
+*/
+const updateOrganisationDetails = function (organisationDetails, token) {
     return new Promise(async (resolve, reject) => {
 
         const updateOrgDetails =
-            process.env.sunbird_url + constants.endpoints.SUNBIRD_UPDATE_ORG;
-        let response = await callToSunbird(token, requestBody, updateOrgDetails ,"PATCH");
+            process.env.SUNBIRD_URL + constants.endpoints.SUNBIRD_UPDATE_ORG;
+        let response = await callToSunbird(token, organisationDetails, updateOrgDetails, "PATCH");
         return resolve(response);
 
-
-    })
+    });
 }
-var getOrganisationDetails = function (requestBody, token) {
+
+/**
+  * To get organisational details
+  * @function
+  * @name getOrganisationDetails
+  * @param organisationDetails - organisation details .
+  * @param token - Logged in user token.
+  * @returns {JSON} - return updated organisation details
+*/
+const getOrganisationDetails = function (requestBody, token) {
     return new Promise(async (resolve, reject) => {
 
         const OrgDetails =
```

Comment:
@rakeshSgr - Variable name has to be a camel case. Please rename it to orgDetailsAPIEndpoint